### PR TITLE
fix [8693f7em4] styles correction for registration and authentication

### PR DESCRIPTION
**Pull request by Svitlana-G**
1. Add links for privacy and terms of service
2. Fix styles for button on page email-sing-in
3. Set the layout height to 200 px
4. Use common styles

### CODE REVIEW

File: src/components/userAuth/createAccount/CreateAccount.tsx [hunk 1]
```diff
@@ -1,19 +1,23 @@
...
 import { Typography, Box, Link } from '@mui/material'
 import { makeStyles } from 'tss-react/mui'
+import { commonStyles } from 'styles/commonStyles'
 
 const CreateAccount = () => {
   const { classes } = useStyles()
+  const commonClasses = commonStyles().classes
   return (
-    <Box className={classes.mainBox}>
+    <Box className={commonClasses.mainBox + ' ' + classes.mainGrid}>
```
Comment (LyubovSugak) [inline src/components/userAuth/createAccount/CreateAccount.tsx:10]: you want to use both classes ?

File: src/components/userAuth/signInMail/SignInMail.js [hunk 2]
```diff
@@ -127,7 +129,8 @@ const SignInMail = () => {
         variant="contained"
         disableElevation
         disableRipple
-        className={classes.loginBtn}
+        disabled={!inputEmail || !inputPassword ? true : false}
```
Comment (LyubovSugak) [inline src/components/userAuth/signInMail/SignInMail.js:132]: smth wierd, maybe you need
`!!inputPassword`

File: src/components/userAuth/createAccount/CreateAccount.tsx (hunk 1 above)
Comment (Svitlana-G) [reply src/components/userAuth/createAccount/CreateAccount.tsx:10]: yes

Comment (LyubovSugak) [reply src/components/userAuth/createAccount/CreateAccount.tsx:10]: className={`${mainBox} ${mainGrid}`}

File: src/components/userAuth/signInMail/SignInMail.js (hunk 2 above)
Comment (Svitlana-G) [reply src/components/userAuth/signInMail/SignInMail.js:132]: The button must be disabled if no password or email is entered. Both fields (email and password) must be filled

Comment (Svitlana-G) [reply src/components/userAuth/signInMail/SignInMail.js]: Thanks, understood. Really unnecessary

**Review by LyubovSugak — APPROVED** [review]

**Pull request merged**